roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -64,13 +64,14 @@
   when:
   - openshift.hosted.registry.storage.kind | default(none) == 'object'
 
-- name: Set facts for persistent volume
+- name: Update openshift_hosted facts for persistent volumes
   set_fact:
+    openshift_hosted_registry_volumes: "{{ openshift_hosted_registry_volumes | union(pvc_volume_mounts) }}"
+  vars:
     pvc_volume_mounts:
     - name: registry-storage
       type: persistentVolumeClaim
```

Comment:
@mtnbikenc The interface can be as we want.  Currently it expects type to be pvc `        elif volume_info['type'] == 'pvc':`.  We would just need to add support for that.

For this to work, it needs to be pvc currently.   I'm fine if we want to add support for both.